src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -6,11 +6,12 @@
 
 (defn collectible-balance
   [collectible]
-  (-> collectible
-      :ownership
-      first
-      :balance
-      js/parseInt))
+  (let [balance (-> collectible
+                    :ownership
+                    first
+                    :balance
+                    js/parseInt)]
```

Comment:
can use `utils.number/parse-int` helper, that already has 0 as a default

Reply:
Sorry i saw it late, as QA already tested i will try to apply it in next PR

Reply:
oh btw `integer?` condition didn't help to catch ##NAN, this was only workaround.

Reply:
@mohsen-ghafouri, if possible, could you share an example where `integer?` doesn't work? `(parse-int js/NaN)` returns the default value 0 as expected.

Reply:
I see, i used `integer?` without using parse-int so it didn't work